src/api/api.controller.ts

```diff
@@ -165,6 +167,24 @@ export class ApiController {
         return await this.issuerService.revokeCredential(body.credential_exchange_id, body.publish);
     }
 
+    /**
+     * Deletes a credential for the issuer
+     */
+    @ApiResponse({ status: 200, type: IssueDeleteResDto })
```

Comment:
You can remove the response DTOs, looks like aca-py just returns an empty object